hudi-common/src/test/java/org/apache/hudi/common/functional/TestHoodieLogFormat.java

```diff
@@ -155,6 +155,7 @@ public void testEmptyLog() throws IOException {
     assertEquals(0, writer.getCurrentSize(), "Just created this log, size should be 0");
     assertTrue(writer.getLogFile().getFileName().startsWith("."), "Check all log files should start with a .");
     assertEquals(1, writer.getLogFile().getLogVersion(), "Version should be 1 for new log created");
+    writer.close();
```

Comment:
@nsivabalan for the future reference: it's easier to manage and more predictable to use try-with-resources statement:

```
try (Closeable c = ...) {
  // Will be closed automatically even in the presence of exceptions
}
```